### ISSUE TITLE
Log specific state with it's action

### ISSUE THREAD

**Issue by fouad1994** [LogRocket/redux-logger#325]
Can I select what state to log?

**Comment by eranimo** [comment]
Yes, you can use the [`stateTransformer`](https://github.com/LogRocket/redux-logger#statetransformer--state-object--state) option.

**Comment by fouad1994** [comment]
I read about it but really I couldn't get know how to use it correctly
